packages/frontend/src/hooks/useSafeNavigation.ts

```
@@ -0,0 +1,198 @@
+  const escapeEvent = new KeyboardEvent("keydown", {
+    key: "Escape",
+    code: "Escape",
+    keyCode: 27,
+    bubbles: true,
+    cancelable: true,
+  });
+
+  // Close any open Select/MenuTrigger popovers
+  document
+    .querySelectorAll('[data-open], button[aria-expanded="true"]')
+    .forEach((el) => el.dispatchEvent(escapeEvent));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)